main.go

```diff
@@ -162,6 +164,62 @@ func dumpJSONFile(file string, pretty bool, schema bool) error {
 	return nil
 }
 
+func getFilesContentHash(files []string, tmp string) (string, bool, error) {
+	sha1 := sha1.New()
+	stdinTmpCopy := false
+
+	for i := 0; i < len(files); i++ {
+		err := func() error {
```

Comment:
It doesn't really help code *structure* to use a nested function literal like this. I'd normally just break it out into a separate top-level function. That normally also makes it easier if you need to unit test it.

Reply:
But you don't have to make that change right now. It's ok for now